### PR TITLE
docs(cdn): adding new/updated documentation on CDN packages

### PR DESCRIPTION
**Pull request by jeffchew**
### Related Ticket(s)

No related issue

### Description

This is updated documentation on using the CDN packages for web components. In addition, this adds a new page for the CDN styles helpers. This page is pulled from the main `Carbon for IBM.com` repo and adapted for the website.

### Changelog

**New**

- Carbon CDN style helpers page

**Changed**

- Updated documentation on web components CDN usage

### CODE REVIEW

File: src/pages/developing/frameworks/web-components.mdx [hunk 1]
```diff
@@ -81,11 +61,80 @@ without the need for front-end bundling:
 </html>
 ```
 
-<InlineNotification>
+To see all available CDN links, visit the `Docs` tab for the corresponding components in [Storybook](https://www.ibm.com/standards/carbon/web-components).
 
-**Note:** Starting in Carbon for IBM.com Web Components v1.6.0, CDN artifacts will be available for all components! More details coming soon!
+### Tag vs. specific version
```
Comment (janchild) [inline src/pages/developing/frameworks/web-components.mdx:66]:
```suggestion
### Tag versus specific version
```

File: src/data/nav-items.yaml [hunk 2]
```diff
@@ -17,6 +17,8 @@
       path: /developing/other-packages
     - title: Building for IBM.com
       path: /developing/building-for-ibm-dotcom
+    - title: Carbon CDN Style Helpers
```
Comment (janchild) [inline src/data/nav-items.yaml:20]:
```suggestion
    - title: Carbon CDN style helpers
```

File: src/pages/developing/carbon-cdn-style-helpers/heading.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Carbon CDN Style Helpers
```
Comment (janchild) [inline src/pages/developing/carbon-cdn-style-helpers/heading.txt:2]:
```suggestion
title: Carbon CDN style helpers
```

**Review by janchild — COMMENTED** [review]
Nice! Just a couple of tweaks. Let me know if you have any questions!

**Comment by jeffchew** [comment]
> Nice! Just a couple of tweaks. Let me know if you have any questions!

Thank you for the review @janchild ! Just committed your suggestions!

**Review by janchild — APPROVED** [review]
🚀

**Pull request merged**